src/optimalcontrol/src/ConstraintsGroup.cpp

```diff
@@ -11,10 +11,13 @@
  */
 
 #include "iDynTree/Core/VectorDynSize.h"
+#include "iDynTree/Core/MatrixDynSize.h"
 #include "iDynTree/Core/Utils.h"
 #include "iDynTree/ConstraintsGroup.h"
 #include "iDynTree/Constraint.h"
 #include "iDynTree/TimeRange.h"
+#include "iDynTree/Core/EigenHelpers.h"
+#include "Eigen/Dense"
```

Comment:
Use `<..>` for external headers (I would use them also for internal headers this file and the headers are not in the same directory, but let's remain consistent for the moment.

Reply:
Would you prefer to use *always* ``<..>`` format?